rules-reviewed/quarkus1/quarkus1-10/tests/data/pom.xml

```diff
@@ -44,7 +44,6 @@
       <artifactId>rest-assured</artifactId>
       <scope>test</scope>
     </dependency>
-    <dependency>
```

Comment:
why ?

Reply:
@m-brophy - Jonathan has picked up on a mistake here. Can you correct it please I'd like to get this merged to master and available to QA on Monday. Thanks